src/benchmarks/micro/libraries/System.Net.Primitives/DnsTests.cs

```diff
@@ -15,5 +16,10 @@ public class DnsTests
 
         [Benchmark]
         public string GetHostName() => Dns.GetHostName();
+
+        private string _hostname = Dns.GetHostName();
+
+        [Benchmark(OperationsPerInvoke = 1000)]
```

Comment:
this benchmarks performs a single operation per invocation, was it a copy-paste error?

```suggestion
        [Benchmark]
```

Reply:
@danmoseley I can see that you have marked as resolved. Was it a miss-click?

Reply:
I forgot to push my fixes 😪